yarn.lock

```diff
@@ -219,10 +219,6 @@ arr-flatten@^1.0.1:
   version "1.1.0"
```

Comment:
sure you want to change this?

Reply:
i actually did not change this.. no idea where this came from

Reply:
(this is actually the same version now, i guess after pull from master it was fixed)